openupgrade_scripts/scripts/web/15.0.1.0/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,41 @@
+---Models in module 'web'---
+---Fields in module 'web'---
+---XML records in module 'web'---
+NEW ir.actions.report: web.action_report_layout_preview
+NEW ir.ui.view: web.assets_backend_legacy_lazy
+NEW ir.ui.view: web.external_layout_bold
+NEW ir.ui.view: web.external_layout_striped
+NEW ir.ui.view: web.preview_layout_report
+NEW ir.ui.view: web.test_helpers
+NEW ir.ui.view: web.view_speedscope_index
+DEL ir.ui.view: web._assets_backend_helpers
+DEL ir.ui.view: web._assets_bootstrap
+DEL ir.ui.view: web._assets_common_minimal_js
+DEL ir.ui.view: web._assets_frontend_helpers
+DEL ir.ui.view: web._assets_frontend_minimal_js
+DEL ir.ui.view: web._assets_helpers
+DEL ir.ui.view: web._assets_primary_variables
+DEL ir.ui.view: web._assets_secondary_variables
+DEL ir.ui.view: web._assets_utils
+DEL ir.ui.view: web.assets_backend
+DEL ir.ui.view: web.assets_backend_prod_only
+DEL ir.ui.view: web.assets_common
+DEL ir.ui.view: web.assets_common_lazy
+DEL ir.ui.view: web.assets_common_minimal_js
+DEL ir.ui.view: web.assets_frontend
+DEL ir.ui.view: web.assets_frontend_lazy
+DEL ir.ui.view: web.assets_frontend_minimal_js
+DEL ir.ui.view: web.assets_tests
+DEL ir.ui.view: web.external_layout_background
+DEL ir.ui.view: web.external_layout_clean
```

Comment:
Can you rename these two XML ids (background/clean) to the new external layout views (bold/striped)? Otherwise the old report layout views are going to be deleted, leaving some companies without an external_report_layout_id.

The mapping should be clear from https://github.com/odoo/odoo/commit/bc40795c56eb463fea29be08306883d7d8ea31fc

Reply:
yes, i will change them